src/OrchardCore.Modules/OrchardCore.Admin/AdminMenuFilter.cs

```diff
@@ -36,7 +36,9 @@ public void OnResultExecuting(ResultExecutingContext filterContext)
                 return;
             }
 
-            if (filterContext.HttpContext.Response.StatusCode != 200)
+            // Don't create the menu if the status code is 3xx
+            var statusCode = filterContext.HttpContext.Response.StatusCode;
+            if (statusCode >= 300 && statusCode < 400)
```

Comment:
Why not just 200?

Reply:
There's an explanation here: https://github.com/OrchardCMS/OrchardCore/issues/1078